crates/dojo-world/src/contracts/naming.rs

```diff
@@ -62,19 +62,16 @@ pub fn compute_bytearray_hash(namespace: &str) -> Felt {
     poseidon_hash_many(&ByteArray::cairo_serialize(&ba))
 }
 
-pub fn compute_model_selector_from_tag(tag: &str) -> Felt {
+pub fn compute_selector_from_tag(tag: &str) -> Felt {
     let (namespace, name) = split_tag(tag).unwrap();
-    compute_model_selector_from_names(&namespace, &name)
+    compute_selector_from_names(&namespace, &name)
 }
 
-pub fn compute_model_selector_from_names(namespace: &str, model_name: &str) -> Felt {
-    compute_model_selector_from_hash(
-        compute_bytearray_hash(namespace),
-        compute_bytearray_hash(model_name),
-    )
+pub fn compute_selector_from_names(namespace: &str, name: &str) -> Felt {
+    compute_selector_from_hash(compute_bytearray_hash(namespace), compute_bytearray_hash(name))
 }
 
-pub fn compute_model_selector_from_hash(namespace_hash: Felt, model_hash: Felt) -> Felt {
+pub fn compute_selector_from_hash(namespace_hash: Felt, model_hash: Felt) -> Felt {
```

Comment:
```suggestion
pub fn compute_selector_from_hashes(namespace_hash: Felt, model_hash: Felt) -> Felt {
```
To make it consistent with `compute_selector_from_names`.